Sources/TuistSupport/Utils/Functions.swift

```diff
@@ -5,8 +5,7 @@
 ///
 /// - Parameter keyPath: A key path.
 /// - Returns: A getter function.
-public func get<Root, Value>(_ keyPath: KeyPath<Root, Value>) -> (Root) -> Value {
-    { root in root[keyPath: keyPath] }
+public func get<Root, Value>(_ keyPath: KeyPath<Root, Value>) -> (Root) -> Value { { root in root[keyPath: keyPath] }
```

Comment:
this formatting looks odd, what version of `swiftformat` are you using?

Reply:
I'm using the latest, 0.44.4 🤔